apps/publications/src/app/app.module.ts

```diff
@@ -57,7 +78,16 @@ const loadConfigs = (appConfig: PublicationsConfigService) => {
     ApiModule,
     HttpClientModule,
     AppRoutingModule,
-    RouterModule.forRoot([], { initialNavigation: 'enabled' }),
+    MatSidenavModule,
+    MatListModule,
+    PerunSharedComponentsModule,
+    MatMenuModule,
+    MatToolbarModule,
+    MatSortModule,
+    MatTableExporterModule,
+    MatPaginatorModule,
+    MatTooltipModule,
```

Comment:
All this Mat-something things that you imported should already be in the  UiMaterialModule and if they are not here, import the Mat-something in that module, I think It would be more clean